pkg/virt-controller/watch/migration.go

```diff
@@ -624,14 +624,17 @@ func (c *MigrationController) createTargetPod(migration *virtv1.VirtualMachineIn
 	key := controller.MigrationKey(migration)
 	c.podExpectations.ExpectCreations(key, 1)
 	pod, err := c.clientset.CoreV1().Pods(vmi.GetNamespace()).Create(context.Background(), templatePod, v1.CreateOptions{})
-	if k8serrors.IsForbidden(err) && strings.Contains(err.Error(), "violates PodSecurity") {
-		psaErr := fmt.Errorf("failed to create target pod for vmi %s/%s, it needs a privileged namespace to run: %w", vmi.GetNamespace(), vmi.GetName(), err)
-		c.recorder.Eventf(vmi, k8sv1.EventTypeWarning, FailedCreatePodReason, failedToRenderLaunchManifestErrFormat, psaErr)
-		return psaErr
-	} else if err != nil {
-		c.recorder.Eventf(vmi, k8sv1.EventTypeWarning, FailedCreatePodReason, "Error creating pod: %v", err)
+	if err != nil {
```

Comment:
I would even move the expectation reset directly under the err check.

Reply:
:+1: I will edit this if a chance comes up.